recipes/viennarna/2.4.7/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "2.4.7" %}
```

Comment:
Do we need to retain the recipe for the older version? Any particular reason?

Reply:
mhh.. I am just updating (since I need the new version)

so far, the responsible people stored the old version's data in a subfolder, that's why I did the same.
ping: @s-will @jfallmann : is explicitely retaining the old version needed?